scraper/scraper.Dockerfile

```diff
@@ -50,4 +50,19 @@ RUN useradd app -u ${APP_UID} \
 # Make it more lightweight by removing tests
 RUN rm -r /opt/scraper/tests
 
+CMD [ "/bootstrap.sh" ]
+
+FROM scraper_prod AS scraper_dev
```

Comment:
This line is used to make it the dev Dockerfile? Should I delete it to use the production version?

Reply:
If you want to use development version, add `target: scraper_dev` to your docker-compose file (check example), otherwise, `target: scraper_prod`. The same goes for `docker build -f scraper.Dockerfile --target <your-target>`